### ISSUE TITLE
The type initializer of 'nspector.Common. Drs ServiceLocator' threw an exception.

### ISSUE THREAD

**Issue by lean13377** [Orbmu2k/nvidiaProfileInspector#132]
The type initializer of 'nspector.Common. Drs ServiceLocator' threw an exception.
em nspector.frmDrvSettings..ctor(Boolean showCsnOnly, Boolean skipScan) em nspector.Program.Main(String[] args)

Please Help me.

**Comment by GregariousJB** [comment]
Same problem here, but this issue is a duplicate: #116

**Comment by Orbmu2k** [comment]
should be fixed in https://github.com/Orbmu2k/nvidiaProfileInspector/releases/tag/2.4.0.4

**Comment by ziadass** [comment]
he type initializer of 'nspector.Common. Drs ServiceLocator' threw an exception. still happening to me

**Comment by Xigua7i** [comment]
> should be fixed in https://github.com/Orbmu2k/nvidiaProfileInspector/releases/tag/2.4.0.4应在 https://github.com/Orbmu2k/nvidiaProfileInspector/releases/tag/2.4.0.4 中修复

It's doesn't working.

**Comment by Noisis** [comment]
can someone help me with this error?
![Screenshot (11)](https://github.com/Orbmu2k/nvidiaProfileInspector/assets/151835466/829e7137-1455-476c-9ce3-8ce6686827d5)

**Comment by charlie2231** [comment]
I accidentally deleted the Fortnite file from the NVIDIA Profile Inspector help

**Comment by CybexBilla** [comment]
> The type initializer of 'nspector.Common. Drs ServiceLocator' threw an exception. em nspector.frmDrvSettings..ctor(Boolean showCsnOnly, Boolean skipScan) em nspector.Program.Main(String[] args)
> 
> Please Help me.

Did your issue got solved?

**Comment by alosasokds** [comment]
![Error 8_13_2024 5_17_20 PM](https://github.com/user-attachments/assets/e162cd62-13a8-4cfd-8338-72dc1b3d0d14)

**Comment by alosasokds** [comment]
qny helep 
any help

**Comment by JAyOPTRA** [comment]
bro is still happening bro what do i do 
![Screenshot 2024-08-19 165542](https://github.com/user-attachments/assets/ff160487-8a05-444c-974e-71ed5fe1daea)

**Comment by JAyOPTRA** [comment]
PLEase help meeee

**Comment by YomamaNow** [comment]
same problem...

**Comment by YomamaNow** [comment]
how is this closed yet never solved? lol

**Comment by Gary9** [comment]
> The type initializer of 'nspector.Common. Drs ServiceLocator' threw an exception. em nspector.frmDrvSettings..ctor(Boolean showCsnOnly, Boolean skipScan) em nspector.Program.Main(String[] args)
> 
> Please Help me.

I had the same problem.
Using Visual Studio 2022, I found out that there is an inner exception that was not shown in the exception message, that I needed permisions. I realized my error as I had opend it from my downloads folder. If you havn't already done so, copy the whole uncompressed folder to your Progam FIles (86) and open it from there. 

If you still get the same error and you have the skills then open it in VS 2002 and run it in debug to find out what the inner exception message is.
